.github/workflows/on_run_publish_surefire_report.yml

```diff
@@ -78,7 +78,7 @@ jobs:
             MIN_TEST_NUM: 30000
             GH_TOKEN: ${{ github.token }}
           run: |
-            COUNT=$(find . -name "TEST-*Test.xml" | while read f; do xmlstarlet sel -t -c  "count(//testcase)" -n $f; done | awk '{s+=$1} END {print s}')
+            COUNT=$(find . -name "TEST-*.xml" | while read f; do xmlstarlet sel -t -c  "count(//testcase)" -n "$f"; done | awk '{s+=$1} END {print s}')
```

Comment:
Won't this also include the `TEST-*FLAKY.xml` files?

Reply:
> Won't this also include the `TEST-*FLAKY.xml` files?

it's no more needed since we exclude those files when we build the surefire-test-report artifact:
[Archive surefire test report](https://github.com/infinispan/infinispan/blob/aa1d202b3c10fddbb6b82d6ef660a2adc78ed3eb/.github/workflows/on_propen_push_do_test.yml#L58-L69)